### ISSUE TITLE
Replace abbr[title] with abbr[aria-label]

### ISSUE THREAD

**Issue by ffoodd** [ffoodd/sseeeedd#85]
`aria-label` has much better support than `abbr[title]` thus should be encouraged.

**Comment by GaetanBt** [comment]
How could we make it usable for users that use their mouse to display the title of the abbreviation ?

**Comment by ffoodd** [comment]
`abbr`? You can't :)
You have to use the tooltip component…
